apps/http_proxylet.go

```diff
@@ -83,39 +84,69 @@ func ConfigureDefaultTransport() {
 	}
 }
 func main() {
-	log.Info("Version %s", pkg.VERSION)
+	log.Infof("Version %s", pkg.VERSION)
 	logLevel := httpproxy.GetEnvWithDefaults("LOG_LEVEL", "debug")
 	level, levelerr := log.ParseLevel(logLevel)
 	if levelerr != nil {
 		log.Infof("No valid log level from ENV, defaulting to debug level was: %s", level)
 		level = log.DebugLevel
 	}
 	log.SetLevel(level)
+	ConfigureDefaultTransport()
 	err := models2.InitNats()
 	if err != nil {
 		log.Fatalf("Unable to init NATS %s", err.Error())
 	}
 	nc := models2.GetNatsClient()
+	//default to listen for everything
+	httpproxy.SetMyLocationID("*")
+	setupQueueSubscriptions()
 	_, err = nc.Subscribe(server.ResponseForLocationID, func(msg *nats.Msg) {
 		locationID := string(msg.Data)
+		if len(locationID)==0{
+			locationID="*"
+		}
 		httpproxy.SetMyLocationID(locationID)
 		log.Infof("Using location ID %s", locationID)
+		setupQueueSubscriptions()
 
 	})
 	if err != nil {
 		log.Fatalf("Unable to talk to NATS %s", err.Error())
 	}
+
+
 	nc.Publish(server.RequestForLocationID, []byte(""))
 
-	ConfigureDefaultTransport()
-	err = models2.InitNats()
-	if err != nil {
-		log.Fatalf("Unable to connect to NATS %s", err.Error())
+
+
+	//if *showTime {
+	//	log.SetFlags(log.LstdFlags)
+	//}
```

Comment:
Is there a reason we're leaving this commented out here?

Reply:
not sure, I can un comment it